deployment/network-operator/templates/admission_controller.yaml

```diff
@@ -0,0 +1,129 @@
+---
+{{- if .Values.admissionController.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-webhook-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
+  selector:
+    control-plane: controller-manager
```

Comment:
This  selector and its corresponding label set in the operator.yaml is WAAAAYYY to generic for this service

Reply:
we can: `conrol-plane: {{ .Release.Name }}-controller`

issue with current: if several controllers are deployed in the same ns.
(i already saw use-cases where netop and gpu op deployed in the same ns)

Reply:
Done